### PR TITLE
[11.0][ADD] cb_number_of_holidays_report

### PR DESCRIPTION
**Pull request by jarroyomorales**

### CODE REVIEW

File: cb_number_of_holidays_report/report/holidays_count_report.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                [('employee_id', '=', employee.id),
+                 ('type', '=', 'remove'),
+                 ('date_from', '<=', date_to),
+                 ('date_to', '>=', date_from),
+                 ('state', '=', 'validate'),
+                 ]
+            )
+            import logging
+            days_count = 0.0
+            date_from_day = fields.Date.from_string(date_from)
```
Comment (etobella) [inline cb_number_of_holidays_report/report/holidays_count_report.py]: Remove it please

File: cb_number_of_holidays_report/wizards/wizard_holidays_count.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+class WizardHolidaysCount(models.TransientModel):
+
+    _name = 'wizard.holidays.count'
+
+    name = fields.Char()
+
+    date_from = fields.Date(required=True)
+    date_to = fields.Date(required=True)
+
+    employee_ids = fields.Many2many('hr.employee')
```
Comment (etobella) [inline cb_number_of_holidays_report/wizards/wizard_holidays_count.py:17]: Fill with populate please

File: cb_number_of_holidays_report/report/holidays_count_report.py (hunk 1 above)
Comment (etobella) [inline cb_number_of_holidays_report/report/holidays_count_report.py]: Remove it ple

**Review by etobella — CHANGES_REQUESTED** [review]

File: cb_number_of_holidays_report/wizards/wizard_holidays_count.py (hunk 2 above)
Comment (jarroyomorales) [reply cb_number_of_holidays_report/wizards/wizard_holidays_count.py:17]: Yes yes, it is WIP

Comment (etobella) [reply cb_number_of_holidays_report/wizards/wizard_holidays_count.py:17]: :wink:

File: cb_number_of_holidays_report/report/holidays_count_report.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                 ('state', '=', 'validate'),
+                 ('count_in_hours', '=', False)
+                 ]
+            )
+
+            days_count = 0.0
+            date_from_day = fields.Datetime.from_string(date_from)
+            date_to_day = fields.Datetime.from_string(date_to)
+            for holiday in holidays:
+                if date_from >= holiday.date_from and (
```
Comment (etobella) [inline cb_number_of_holidays_report/report/holidays_count_report.py:40]: you should use https://github.com/odoo/odoo/blob/11.0/addons/hr_holidays/models/hr_holidays.py#L284

**Review by etobella — CHANGES_REQUESTED** [review]

File: cb_number_of_holidays_report/report/holidays_count_report.py (hunk 3 above)
Comment (jarroyomorales) [reply cb_number_of_holidays_report/report/holidays_count_report.py:40]: Yes, I guess it's more clear, I can set employee to false and use this.

**Comment by jarroyomorales** [comment]
@etobella All comments attended, added translation and fixed tests. I think this module is done.

**Comment by jarroyomorales** [comment]
@etobella Coverage subido, no sé porque no se actualiza el comment de codecov...

**Comment by jarroyomorales** [comment]
https://codecov.io/gh/tegin/cb-addons/tree/b187a6b0a42870f0bdce0c6e8e6f442ee85c4177/cb_number_of_holidays_report
Aqui se puede ver que el coverage es de 96,97%, se ha quedado pillado...

**Comment by jarroyomorales** [comment]
Replaced by https://github.com/tegin/cb-hr/pull/2

**Pull request closed without merging**